lib/flowcommerce_spree.rb

```diff
@@ -12,6 +12,9 @@
 require 'request_store'
 
 module FlowcommerceSpree
+  API_KEY = ENV.fetch('FLOW_TOKEN', 'test_key')
```

Comment:
Oops, this won't work - I remember I've told about ENV.fetch not working on a class definition level.

That's why I have moved such fetching into the engine's `before_initialize` hook - https://github.com/mejuri-inc/flowcommerce_spree/blob/no-ticket-flow-token-default/lib/flowcommerce_spree/engine.rb#L9

Reply:
:facepalm: so sorry. I completely forgot about this as well. 

The weird thing is that it worked ok when I tested (I tested it in one of the devX environments), but not sure why. I'd rather have it the same way you did. 

I will finish writing a little spec I am working on right now, and will do this change. After that will test it and merge it to avoid forgetting about these later. Thanks for the heads up!!

Reply:
@texpert funny thing. I tried deploying this branch without this change and it is not working properly. It is raising the error of the FLOW_TOKEN not defined. Adding it here it helps. I think that for now I would keep it in both files (here and in the flowcommerce_spree/engine.rb) to solve all possible issues. What do you think?

Reply:
@sebastiandl , no need to keep it in both places. I've searched thoroughly in RubyMine all the gems installed with our mejuri-web and found nowhere ENV.fetch to be used on a class level. Found only one place in mejuri-web - `app/services/ambassador_manager.rb1 - wonder is it working or not.

Weird thing is I haven't also found mentions of this behaviour on internet.

Reply:
Ohh. Yep the only references I searched for this was related to how this ENV.fetch works, actually. So for now then I will  remove the changes made on this Pr for https://github.com/mejuri-inc/flowcommerce_spree/blob/no-ticket-flow-token-default/lib/flowcommerce_spree/engine.rb#L9 and retest to be sure it does no break hehe. Thanks for the help and taking the time to look into this Aurel!